x-pack/plugins/cases/common/api/cases/comment.ts

```diff
@@ -27,6 +28,7 @@ export const CommentAttributesBasicRt = rt.type({
   ]),
   created_at: rt.string,
   created_by: UserRT,
+  owner: rt.string,
```

Comment:
I'm wondering if we should make a type for owner like:
```
const OwnerTypeRT = rt.union([rt.literal('securitySolution'), rt.literal('observability')]);
```
Not for this PR, but I think it would be good to have strict control over that type and not just accept any old string.